testing/cpm/cpm_nvcomp-override-clears-proprietary_binary.cmake

```diff
@@ -0,0 +1,46 @@
+#=============================================================================
+# Copyright (c) 2021, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#=============================================================================
+include(${rapids-cmake-dir}/cpm/init.cmake)
+include(${rapids-cmake-dir}/cpm/nvcomp.cmake)
+include(${rapids-cmake-dir}/cpm/package_override.cmake)
+
+
+rapids_cpm_init()
+
+if(TARGET nvcomp::nvcomp)
+  message(FATAL_ERROR "Expected nvcomp::nvcomp expected to not exist")
+endif()
+
+# Need to write out an nvcomp override file
+file(WRITE ${CMAKE_CURRENT_BINARY_DIR}/override.json
```

Comment:
Should we clean up the file after the test completes?

Reply:
This is a pattern I learned from CMake. We keep the test dir in a 'dirty' state so we have an exact record of what was executed. Helps diagnose issues since we can upload an exact state of a failure if needed

Reply:
Ah I see.